modules/reference/pages/configuration-parameters.adoc

```diff
@@ -1530,4 +1530,13 @@ a|
 * The default value is 0. When the value is 0, the threshold is dynamically decided by TigerGraph, based on current memory usage statistics.
 | `TransactionSizeLimit=1073741824`
 
+|GSE
+| `ServFromDisk`,
+GSE.BasicConfig.Env
+a|
+* Enable the disk mode to read vertex ID from the disk instead of memory by setting it to `1` (Default = `0`). This will help reduce the GSE startup time significantly by skipping loading vertices into the memory from the disk with the trade-off of slight (< 10%) performance degradation for data loading and query processing.
```

Comment:
```suggestion
* When set to `1` (default = '10`), vertex ID data remains on disk rather than being loaded into memory. This will help reduce the GSE startup time significantly, with the trade-off of slight slower performance for data loading and query processing.
```